File: tests/cases/delete.py
#!/usr/bin/env python3


def show():
    # Delete from list
    my_list = [1, 2, 3, 4, 5]
    del my_list[2]
    print(len(my_list))

    # Delete from dict
    my_dict = {"a": 1, "b": 2, "c": 3}
    del my_dict["b"]
    print(len(my_dict))


if __name__ == "__main__":
    show()

File: tests/expected/delete.kt
fun show() {
    var my_list = arrayOf(1, 2, 3, 4, 5)
// del unimplemented on line 7:4
    if (true) {
        val __tmp1 = my_list.size
        println("$__tmp1")
    }
    var my_dict = hashMapOf("a" to 1, "b" to 2, "c" to 3)
// del unimplemented on line 12:4
    if (true) {
        val __tmp2 = my_dict.size
        println("$__tmp2")
    }
}

fun main(argv: Array<String>) {
    show()
}
